index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>portfolio</title>
+  </head>
+  <body>
+    <header class="nav">
+      <a href="#">john doe</a>
+      <i class="fa-solid fa-bars"></i>
+    </header>
+    <section   class="main">
+      <h1>Hey There. I`m Nick</h1>
+      <h5>i am a software developer</h5>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <ul>
+        <li>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-solid fa-m"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-solid fa-m"></i></a>
+        </li>
+      </ul>
+    </section>
+  </body>
+</html>
+
```

Comment:
- [ ] As the previous reviewer suggested, Try to resemble your design similar to the one provided by Figma design. I can see  that you 've tried to make some changes, but again you didn't use fonts, and spacing similar to the one provided in the design. try to Adjust your font weight, family, size, etc. so that you can have the design similar or much more closer to the one provided from the design. 

Your design | Figma design
--- | ---
![image](https://user-images.githubusercontent.com/43877689/170700680-c1328313-3ff3-4794-9d7e-88aebe410381.png) | ![image](https://user-images.githubusercontent.com/43877689/170701343-2735d891-4a38-43d4-af6d-1c8fa2c46acf.png)